test/e2e/data/infrastructure-vsphere-govmomi/main/multi-disk/data-disks-patch.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
+kind: VSphereMachineTemplate
+metadata:
+  name: '${CLUSTER_NAME}'
+  namespace: '${NAMESPACE}'
+spec:
+  template:
+    spec:
+      dataDisks:
+      - name: "etcd"
+        sizeGiB: 10
+      - name: "container-images"
+        sizeGiB: 20
```

Comment:
Question: in test these disks won't actually get mounted right? They would just get added to the VM?

If so, let's rename them to something which does not make it seem like this is a full example :-) (just give it some more random names)

Reply:
Sure, I can change the names to generics.